fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2685,6 +2685,16 @@ public static boolean isNotCloudMode() {
 
     @ConfField(mutable = true, masterOnly = false)
     public static String security_checker_class_name = "";
+
+    // the white list for the s3 load endpoint, if it is empty, no white list will be set
+    // for example: s3_load_endpoint_white_list=a,b,c
+    @ConfField(mutable = true)
+    public static String[] s3_load_endpoint_white_list = {};
```

Comment:
Use same name as `jdbc_driver_secure_path`.
And does this config need to be mutable?

Reply:
It needs to be compatible with the existing warehouse, this should be discussed